src/components/Code/Code.js

```diff
@@ -27,7 +28,7 @@ const Code = ({ code, language, highlight, additionalClasses, ...props }) => {
         >
             <pre>
                 <code
-                    dangerouslySetInnerHTML={{ __html: CodeToDisplay }}
+                    dangerouslySetInnerHTML={{ __html: sanitize(CodeToDisplay) }}
```

Comment:
With this configuration (sanitize, dangerouslySetInnerHTML) i cant make block with html code snippet, for example
```
<p>This is how we define paragraph</p>
<script>console.log('This is how we include scripts')</script>
```

Reply:
yes, of course, sorry about that, give ma a second to fix this

Reply:
![Flotiq-components-Code-Base-⋅-Storybook](https://user-images.githubusercontent.com/645716/154471882-0f6b22ae-9f4b-4d2b-a50a-7f8074d864cf.png)
![image (28)](https://user-images.githubusercontent.com/645716/154471885-1e8fa9f4-bf1c-43ed-966b-2de6f8cf87f5.png)